src/Microsoft.ML.DataView/DataViewRowCursor.md

```diff
@@ -1,28 +1,28 @@
 ﻿# `ICursor` Notes
 
 This document includes some more in depth notes on some expert topics for
-`ICursor` implementations.
+`DataViewRow` and `DataViewRowCursor` implementations.
```

Comment:
>implementations [](start = 38, length = 15)

classes?

Reply:
(since it's not an interface anymore I mean. But maybe implementations is still good).

---
In reply to: [272296802](https://github.com/dotnet/machinelearning/pull/3195#discussion_r272296802) [](ancestors = 272296802)

Reply:
Yeah, these aren't interfaces any more -- looks like in .NET BCL docs, they seem to prefer the phrase "derived classes," to describe the behavior of subclasses of some abstract base class, so I'll go with that.

Reply:
OK, so I changed it to derived classes when talking about the classes, but when I was talking about implementation in other contexts to mean literally "how a method is implemented," I kept it in that sense obviously. Plus of course this document still talks about implementation of `IDataView`, which is totally appropriate. Let me know if you like.